service/controller/catalog/resource/appcatalogentry/resource.go

```diff
@@ -74,15 +74,15 @@ func (r Resource) Name() string {
 	return Name
 }
 
-func (r *Resource) getCurrentEntryCRs(ctx context.Context, cr v1alpha1.AppCatalog) (map[string]*v1alpha1.AppCatalogEntry, error) {
-	r.logger.Debugf(ctx, "getting current appcatalogentries for appcatalog %#q", cr.Name)
+func (r *Resource) getCurrentEntryCRs(ctx context.Context, cr v1alpha1.Catalog) (map[string]*v1alpha1.AppCatalogEntry, error) {
+	r.logger.Debugf(ctx, "getting current appcatalogentries for catalog %#q", cr.Name)
 
 	currentEntryCRs := map[string]*v1alpha1.AppCatalogEntry{}
 
 	lo := metav1.ListOptions{
 		LabelSelector: fmt.Sprintf("%s=%s,%s=%s", label.ManagedBy, key.AppCatalogEntryManagedBy(project.Name()), label.CatalogName, cr.Name),
 	}
-	entries, err := r.k8sClient.G8sClient().ApplicationV1alpha1().AppCatalogEntries(metav1.NamespaceDefault).List(ctx, lo)
+	entries, err := r.k8sClient.G8sClient().ApplicationV1alpha1().AppCatalogEntries("").List(ctx, lo)
```

Comment:
```suggestion
	entries, err := r.k8sClient.G8sClient().ApplicationV1alpha1().AppCatalogEntries(metav1.NamespaceAll).List(ctx, lo)
```

Forgot suggestion 🤦‍♂️

Reply:
Great! I never expected there would be a variable for "".

Reply:
It's client-go magic there is a constant for everything. 😄 

I just prefer it as it tells you why we're using empty string.